Formula/ibazel.rb

```diff
@@ -0,0 +1,63 @@
+class Ibazel < Formula
+  desc "Tools for building Bazel targets when source files change"
+  homepage "https://github.com/bazelbuild/bazel-watcher"
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.16.2.tar.gz"
+  sha256 "a927520e7ab3a1fb749043e543c57bb211666cd627d053fbe0e8245730beee75"
+  license "Apache-2.0"
+
+  depends_on "bazel" => [:build, :test]
+  depends_on xcode: :build
```

Comment:
I noticed that the codebase is mostly written in Go. Does that mean this downloads its own Go toolchain at build-time in order to compile `ibazel`?

Reply:
Yes. I just removed that requirement at test time, but to do it at build time would require a patch.

Reply:
Well, if it downloads it's own go it won't meet https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Yes, not really a fan of downloading its own toolchain. I know that's par for the course with Bazel (and supposedly one of its selling points), but we've encountered this behaviour in existing formulae and they've turned out to be a nightmare to maintain.

This formula is probably better suited for an external tap. Thanks for your work here anyway.

Reply:
Would it be acceptable to remove the Go toolchain download with a small (two-line) patch?

If there are existing formula where the bazel toolchain download is a problem, I could also look at refactoring those. Happy to discuss further.

Reply:
Will it not download any duplicate toolchains then?

Reply:
Yep, `go_host_sdk` will use the host toolchain:
https://github.com/bazelbuild/rules_go/blob/master/go/toolchains.rst#go_host_sdk

Reply:
Fixing it to use a system toolchain is good, but I’m wary of carrying around a patch indefinitely. Could you turn this into something configurable when invoking Bazel and submit it upstream?